src/ecpubkey.js

```diff
@@ -8,8 +8,9 @@ var Address = require('./address')
 var ecurve = require('ecurve')
 var secp256k1 = ecurve.getCurveByName('secp256k1')
 
-function ECPubKey(Q, compressed) {
-  if (compressed === undefined) compressed = true
+function ECPubKey (Q, compressed) {
+  if (compressed === undefined)
+    compressed = true
```

Comment:
Agreed, this is being rectified in standard 3.0.0
On 1 Mar 2015 3:12 pm, "Wei Lu" <notifications@github.com> wrote:

> In src/ecpubkey.js
> <https://github.com/bitcoinjs/bitcoinjs-lib/pull/364#discussion_r25563885>
> :
>
> > @@ -8,8 +8,9 @@ var Address = require('./address')
> >  var ecurve = require('ecurve')
> >  var secp256k1 = ecurve.getCurveByName('secp256k1')
> >
> > -function ECPubKey(Q, compressed) {
> > -  if (compressed === undefined) compressed = true
> > +function ECPubKey (Q, compressed) {
> > +  if (compressed === undefined)
> > +    compressed = true
>
> This could be dangerous.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoinjs/bitcoinjs-lib/pull/364/files#r25563885>.
>